app/src/main/res/layout/preference_rich_list_option.xml

```diff
@@ -14,8 +14,8 @@
     <RadioButton
         android:id="@+id/button"
         style="?android:attr/radioButtonStyle"
-        android:layout_width="wrap_content"
-        android:layout_height="match_parent"
+        android:layout_width="32dp"
+        android:layout_height="32dp"
```

Comment:
Using `wrap_content` for both should accomplish the same while being compatible with sizing changes in the radio button.